docs/docs/50-user-guide/70-annotations.md

```diff
@@ -0,0 +1,74 @@
+| `kargo.akuity.io/description` | Any | Free-form string | Human-readable description displayed in the UI. |
+| `kargo.akuity.io/color` | `Stage` | Hex color code (e.g. `#ff8800`) | Optional cosmetic color used in UI. |
+| `kargo.akuity.io/create-actor` | All | Actor identity (username/service) | Injected by the control plane to indicate who created the resource. |
+| `kargo.akuity.io/authorized-stage` | `Freight`, `Promotion`, etc. | `<project>:<stage>` | Indicates which Stage is authorized to manage the resource. |
+| `kargo.akuity.io/promotion` | `PromotionTask`, internal resources | Promotion ID or name | Links the resources to a promotion process. |
+| `kargo.akuity.io/argocd-context` | `Stage` | JSON or internal identifier | Records Argo CD application context from the last promotion. |
+
+:::warning
+Avoid setting system-managed annotations manually unless you're explicitly debugging internal behavior.
+:::
+
+### RBAC Annotations
+
+| Annotation key | Target resource(s) | Possible values | Description |
+|----------------|--------------------|------------------|-------------|
+| `rbac.kargo.akuity.io/managed` | RBAC resources | `"true"` | Marks a resource as managed by Kargo's RBAC system. |
+| `rbac.kargo.akuity.io/claim.<name>` | RBAC resources | Claim name as suffix | Maps an OIDC claim to a policy. |
+
+### Event Annotations
+
+The following annotations are added internally by Kargo's eventing subsystem for tracking and audit. They are **not user-settable**.
+
+| Annotation key | Target resource(s) | Possible values | Description |
+|----------------|--------------------|------------------|-------------|
+| `event.kargo.akuity.io/actor` | Events | Actor string | Actor that triggered the event. |
+| `event.kargo.akuity.io/project` | Events | Project name | Associated project. |
+| `event.kargo.akuity.io/promotion-name` | Events | Promotion name | Related promotion. |
+| `event.kargo.akuity.io/promotion-create-time` | Events | Timestamp | Creation time of the promotion. |
+| `event.kargo.akuity.io/freight-*` | Events | Varies | Additional metadata about Freight. |
+| `event.kargo.akuity.io/stage-name` | Events | Stage name | Target Stage. |
+| `event.kargo.akuity.io/analysis-run-name` | Events | Name | Linked Argo CD `AnalysisRun`. |
+| `event.kargo.akuity.io/verification-*` | Events | Timestamps/status | Tracks verification lifecycle. |
+| `event.kargo.akuity.io/applications` | Events | Comma-separated names | Applications involved in promotion. |
+
+## Labels
+
+| Label key | Target resource(s) | Possible values | Description |
+|-----------|--------------------|------------------|-------------|
+| `kargo.akuity.io/alias` | `Freight` | User-defined string | Friendly identifier for the Freight version. |
+| `kargo.akuity.io/stage` | `Freight`, `Promotion`, etc. | Stage name | Indicates the associated Stage. |
+| `kargo.akuity.io/project` | All | Project name | Denotes Project ownership. |
```

Comment:
That is not what this label does. This label should be applied to Project namespaces with a value of `true` if they are created in advance (not by Kargo), so that the namespace can be "adopted" by a Project having the same name. This is covered in more detail [here](https://docs.kargo.io/user-guide/how-to-guides/working-with-projects#namespace-adoption). Please read up on it to improve the accuracy of this row and you can also link to it from here.